### PR TITLE
[Homework5]: Created logger

### PR DESCRIPTION
**Pull request by DmtrVoloshyn**

### CODE REVIEW

File: Homework/Homework5/Actions.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿namespace Homework5
+{
+    public class Actions
+    {
+
+        public static Result FirstMethod()
+        {
+            Logger.Instance.Log(LogLevel.Info, "Start method: FirstMethod");
+            return new Result();
```
Comment (PeterChernykh) [inline Homework/Homework5/Actions.cs:9]: For what reason we need emptu Result then?

File: Homework/Homework5/Services/LoggerService.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
+﻿using System.Text;
+using Homework5;
+
+class Logger
+{
+    private static Logger? instance;
+    private string received_log = "";
```
Comment (PeterChernykh) [inline Homework/Homework5/Services/LoggerService.cs:7]: What a snake case?

File: Homework/Homework5/Services/LoggerService.cs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            WriteLogsToFile();
+            Console.WriteLine("\nLogs saved.");
+        }
+        else
+        {
+            Console.WriteLine("\nLogs will not be saved.");
+        }
+    }
+
+    private void WriteLogsToFile()
```
Comment (PeterChernykh) [inline Homework/Homework5/Services/LoggerService.cs:57]: We can use SaveLog naming

File: Homework/Homework5/Services/LoggerService.cs [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+        if (confirmationResult.Key == ConsoleKey.Y)
+        {
+            WriteLogsToFile();
+            Console.WriteLine("\nLogs saved.");
+        }
+        else
+        {
+            Console.WriteLine("\nLogs will not be saved.");
+        }
```
Comment (PeterChernykh) [inline Homework/Homework5/Services/LoggerService.cs:54]: We can move thid to a separate method that will return true or false and name it as IsLogToFile(); this methid will check whether we need to save logs into files in this case we will be able to create smth like this: if(IsLogToFile()) { WriteLogsToFile();  }

**Comment by PeterChernykh** [comment]
Very good

File: Homework/Homework5/Actions.cs (hunk 1 above)
Comment (DmtrVoloshyn) [reply Homework/Homework5/Actions.cs:9]: This Class has no logic :( Because I don't clearly understand required

**Pull request merged**